platform/podjs/src/index.ts

```diff
@@ -1 +1,2 @@
 export * from "./browserPod";
+export * from "./manifest-parser/src/manifest";
```

Comment:
Not totally sure anyone else is going to use this, so probably no need to export it.